File: src/main/java/com/uwetrottmann/trakt5/entities/Person.java
package com.uwetrottmann.trakt5.entities;

import org.threeten.bp.LocalDate;

public class Person {

    public String name;
    public PersonIds ids;

    // extended info
    public String biography;
    public LocalDate birthday;
    public LocalDate death;
    public String birthplace;
    public String homepage;

}

File: src/main/kotlin/com/redissi/trakt/entities/Person.kt
package com.redissi.trakt.entities

import com.squareup.moshi.JsonClass
import java.time.LocalDate

@JsonClass(generateAdapter = true)
class Person(
    val name: String? = null,
    val ids: PersonIds? = null,
    // extended info
    val biography: String? = null,
    val birthday: LocalDate? = null,
    val death: LocalDate? = null,
    val birthplace: String? = null,
    val homepage: String? = null
)